emva1288/unittests/test_results.py

```diff
@@ -300,3 +302,66 @@ def test_nans(self):
         self.assertIs(r.DSNU1288_DN(), np.nan)
 
         del r
+
+
+class TestRoutines(unittest.TestCase):
+
+    def setUp(self):
+        # instantiate a bayer filter and a wrong filter
+        bayer_filter = get_bayer_filter(0, 1, 1, 0, 1000, 1000)
+        wrong_filter = np.tile([[1, 1, 1, 0], [1, 0, 0, 1]], (500, 250))
+
+        # Source images. imgc is for color image (image with a bayer filter)
+        self.img = np.abs(np.random.random([1000, 1000]))
+        self.imgc = np.ma.array(self.img, mask=bayer_filter)
+
+        # Image wrong's purpose is to assert that a ValueError is raised
+        self.img_wrong = np.ma.array(self.img, mask=wrong_filter)
+
+        # Pass the highpass filter on the source images
+        res = high_pass_filter(self.img, 5)
+        self.img_filt = res['img']/res['multiplicator']
+        res = high_pass_filter(self.imgc, 5)
+        self.imgc_filt = res['img']/res['multiplicator']
+
+        # To assert that the highpass filter does its job, a low frequency
+        # signal will be introduced in the image
+        signal = np.sin(np.linspace(0, np.pi, 1000))
```

Comment:
this signal has amplitud 1, multiply it by 10 or 50 to make a more realistic

Reply:
done (multiplied by 10). Given the fact that we generate an image which values are between 0 and 1, this gives a signal that is 10e1 higher than the desired image